src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -79,6 +79,37 @@ function parseCertifier(transformedData) {
   };
 }
 
+function getPrimaryContact(data) {
+  // If a certification name is present, we know the form was filled by
+  // a third party or the sponsor, and that they should be primary contact.
+  const useCert =
+    data?.certification?.firstName && data?.certification?.firstName !== '';
+
+  // Depending on the result of useCert, grab the first and last name, phone,
+  // and email from either the `certification` object or the first applicant,
+  // then return so we can set up the `primaryContactInfo` for the backend
+  // notification API service.
+  return {
+    name: {
+      first:
+        (useCert
+          ? data?.certification?.firstName
+          : data?.applicants?.[0]?.full_name?.first) ?? false,
+      last:
+        (useCert
+          ? data?.certification?.lastName
+          : data?.applicants?.[0]?.full_name?.last) ?? false,
+    },
+    email:
+      (useCert ? data?.certification?.email : data?.applicants?.[0]?.email) ??
+      false,
+    phone:
+      (useCert
+        ? data?.certification?.phone_number
+        : data?.applicants?.[0]?.phone_number) ?? false,
+  };
```

Comment:
Every key will either be the data we want, or `false` if it was missing (for instance, email will be missing until we add it to the certification/sponsor sections later).